sway_libs/src/nft/nft_storage.sw

```diff
@@ -6,7 +6,7 @@ pub const ADMIN: b256 = 0x100000000000000000000000000000000000000000000000000000
 pub const APPROVED: b256 = 0x2000000000000000000000000000000000000000000000000000000000000000;
 pub const BALANCES: b256 = 0x3000000000000000000000000000000000000000000000000000000000000000;
 pub const MAX_SUPPLY: b256 = 0x4000000000000000000000000000000000000000000000000000000000000000;
-pub const META_DATA: b256 = 0x5000000000000000000000000000000000000000000000000000000000000000;
+pub const METADATA: b256 = 0x5000000000000000000000000000000000000000000000000000000000000000;
```

Comment:
I would consider renaming this TOKENMETADATA to be consistent with the rest of the code